.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -32,13 +32,19 @@
 
 
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
-## Checklist:
+## Checklist
 - [ ] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** guidelines and confirm that my code follows the code style of this project.
 - [ ] Tests for the changes have been added (for bug fixes / features)
-- [ ] Docs have been added / updated (for bug fixes / features)
 
 
 
-<!-- Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such). -->
+<!-- Mark this to let us know if the docs require adjustments for this PR. -->
+<!-- We have our documentation in a seperate repo, feel free to update the docs accordingly. :) -->
+#### Documentation
+- [ ] Docs need to be added/ updated (for bug fixes / features)
+
+
+
+<!-- Put `closes #XXXX` of `fixes #XXXX` in your comment to auto-close the issue that your PR fixes (if such). -->
```

Comment:
"of" => "or"

Reply:
Smashing in some dutch words hehe 😄 Good catch!